amqp/connection.py

```diff
@@ -292,6 +297,21 @@ def _setup_listeners(self):
             spec.Connection.CloseOk: self._on_close_ok,
         })
 
+    def dispatch_method(self, method_sig, payload, content):
```

Comment:
ain't the dispatch method is same in both places?

Reply:
They are not but the diff is not huge. They differ in methods that should be handled during closing:
```python
        if self.is_closing and method_sig not in (
            spec.Connection.Close, spec.Connection.CloseOk
        ):
```
vs. 
```python
        if self.is_closing and method_sig not in (
            spec.Channel.Close, spec.Channel.CloseOk
        ):
```

Reply:
Maybe we can add list of method not ignored in channel and connection and move the logic to abstract channel. What do you think?

Reply:
sensible that would be